### PR TITLE
=cdd #17851 Mark distributed data as experimental

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-distributed-data/build.sbt [hunk 1]
```diff
@@ -3,6 +3,8 @@ import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
 
+AkkaBuild.experimentalSettings
```
Comment (patriknw) [inline akka-distributed-data/build.sbt]: is this correct? or does this overwrite the above defaultSettings?

**Comment by ktoso** [comment]
Thanks Patrik!

File: akka-distributed-data/build.sbt (hunk 1 above)
Comment (ktoso) [inline akka-distributed-data/build.sbt]: Yes, it's correct - just adds some text to the description

**Comment by patriknw** [comment]
Refs #17851

**Pull request merged**